### DISCUSSION TITLE
For the love of god make the UI follow the fluent design principles

### DISCUSSION THREAD

**Discussion by itayasaf1990xd** [Devolutions/UniGetUI#1635] [General]
For the love of god make the UI follow the fluent design principles

**Comment by Spacellary** [comment]
Which parts specifically do you want to see improved?

Let's try to provide constructive feedback here, please.

**Reply by itayasaf1990xd** [reply]
Come on do not play this game and act like you do not know what I am talking about because we both know the UI is not based on the fluent design components for example:

- Search box wrong shape and wrong blue color for dark/light mode and no animation.
![image](https://github.com/marticliment/WingetUI/assets/89840492/23ce5e98-2530-4862-b315-07c31e887804)

- No Hamburger menu and the whole app layout does not match the guidelines.
For example, the 3 Navigation does not even have the underline indication.

- App dark mode/light mode both do not follow the color scheme (the black color selected is wrong based on Fluent Design guidelines)
For example: top (Windows Photos App) bottom WingetUI
![image](https://github.com/marticliment/WingetUI/assets/89840492/6a57b721-e5d5-4d86-b25c-24fc52c73fd3)

- Some icons have text some do not... the UI should be dynamic and use more/less space accordingly

- Release Note should not open GitHub inside WingetUI.
you should check DevToy a 3rd party and see how his buttons/UI/animation etc look and how the app feels compared to WingetUI.

**Reply by Shak1990** [reply]
It is because the developer uses Python for the GUI while trying to mimic the fluent design UI.
I do not think Python supports WinUI 3 components

**Comment by marticliment** [comment]
Hello @itayasaf1990xd,

I would be pleased to see a Pull Request from you with a completely brand new interface using WinUI3, a technology incompatible (Yes, [this](https://www.reddit.com/r/Python/comments/184vp8g/winui_3_with_python/) does not work well enough) with Python. Meanwhile, I would appreciate that you don't criticize other's work an effort in such a destructive way. 

I (the maintainer) have done my best (and so have done the contributors, translators, testers and other users who help the development of this piece of software), and trust me, I don't try to _fool_ you in any way, you are not paying for WingetUI, fel free to uninstall it if you don't like it. No one forces you to use it. There are other alternatives to WingetUI.

Moreover, the two messages that you have posted are against the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md). Sarcastic expressions such as the ones used on the title, on the first message or on the second one will not be tolerated, especially when directed to users who tried to tried to politely request for specific examples after your first post. **Respecting eachother is mandatory here**.

If you have any other suggestion, improvement, issue report or any other helpful information that you would like to share in a polite, non-harmful, constructive manner, make sure to open a new support ticket or to share them right here. You will be listened and I will do my best to help you.

If you continue posting rude messages or other non-adequate content, your access to this forum will be restricted, and you may even get banned.
